csharp_dotnetcore/3.Welcome-Users/readme.md

```diff
@@ -0,0 +1,47 @@
+﻿
+This sample shows how to create a simple echo bot that welcomes user when they join the conversation. The welcoming pattern shown in this bot is applicable for peronsal (1:1) conversation with bots.
+
+
+## To try this sample
+- Clone the repository
+```bash
+git clone https://github.com/Microsoft/botbuilder-samples.git
+```
+ ### Visual studio
+- Navigate to the samples folder (`BotBuilder-Samples\csharp_dotnetcore\3.Welcome-Users`) and open 3.Welcome-Users.sln in Visual studio 
+- Hit F5
+ ### Visual studio code
+- Open `BotBuilder-Samples\csharp_dotnetcore\3.Welcome-Users` folder
+- Bring up a terminal, navigate to BotBuilder-Samples\csharp_dotnetcore\3.Welcome-Users
+- Type 'dotnet run'.
+
+# Testing the bot using Bot Framework Emulator
+Microsoft Bot Framework Emulator is a desktop application that allows bot developers to test and debug their bots on localhost or running remotely through a tunnel.
+
+- Install the Bot Framework emulator from here
```

Comment:
Install emulator from where? This should be the [https://aka.ms/botframeworkemulator](https://aka.ms/botframeworkemulator) link, I suspect.